examples/tree.rs

```diff
@@ -0,0 +1,61 @@
+extern crate turtle;
+
+use turtle::*;
```

Comment:
This is a smaller detail, but I would prefer if this was `use turtle::{Turtle, Point, Color}` instead of the wildcard import. See below for why you'll need to import `Point`.